physics/lsm_ruc.F90

```diff
@@ -787,6 +790,19 @@ subroutine lsm_ruc_run                                            & ! inputs
         frpcpn = .false.
       endif
 
+      do j  = 1, 1    ! 1:1
```

Comment:
It's probably a good idea to parameterize this "1", used here and below for the "j" loop extent. Also in the declaration of fields that have a "j" dimension.
For now it could be defined in this module, but eventually could be provided as an input from the host.

Reply:
1,1 is changed to jms,jme.